docs/source/public_server.rst

```diff
@@ -36,9 +36,9 @@ This document describes how you can
 
 
 
-.. _ZeroMQ: http://zeromq.org
+.. _ZeroMQ: https://zeromq.org
 
-.. _Tornado: http://www.tornadoweb.org
+.. _Tornado: https://www.tornadoweb.org
```

Comment:
These two links cannot be accessed via https

Reply:
Oh. I'm sorry, I'll revise it and send you a PR.

Reply:
It looks like you've worked this out already, but there's no need to make a new PR - when you push on the same branch, this PR updates automatically.